frontend/src/pages/RequestorPage/RequestorPage.tsx

```diff
@@ -109,6 +110,16 @@ export const RequestorPage = () => {
     selectedRoute,
   });
 
+  const formRef = useRef<HTMLFormElement>(null);
+
+  useHotkeys("mod+enter", () => {
+    if (formRef.current) {
+      formRef.current.requestSubmit();
+    }
+  }, {
+    enableOnFormTags: ["input"],
```

Comment:
need to specifically enable on form tags so that it works when the input is focused